pipeline/inputs/node-exporter-metrics.md

```diff
@@ -65,6 +71,7 @@ The following table describes the available collectors as part of this plugin. A
 | time      | Exposes the current system time.                                                                 | Linux  | v1.8    |
 | uname     | Exposes system information as provided by the uname system call.                                 | Linux  | v1.8    |
 | vmstat    | Exposes statistics from `/proc/vmstat`.                                                          | Linux  | v1.8.2  |
+| systemd collector | Exposes statistics fr systemd.                                                           | Linux  | v1.8.2  |
```

Comment:
I think this is more recent than 1.8.2? 2.1.4?

Reply:
Systemd collector was introduced in 2.1.3:
https://github.com/fluent/fluent-bit/commit/ab60c002ae9edae6548ee129f9f38cecf7732239